### PR TITLE
Catch file transfer error and stop transfer ownership command

### PR DESCRIPTION
**Pull request by ChristophWurst**
Ref https://github.com/nextcloud/server/issues/15664

This does not check quota as I do not have a clue how to do this. But it checks the returned value of the move operation. Better than nothing.

### CODE REVIEW

File: apps/files/lib/Command/TransferOwnership.php [hunk 1]
```diff
@@ -151,7 +151,11 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 		$this->collectUsersShares($output);
 
 		// transfer the files
-		$this->transfer($output);
+		if ($this->transfer($output) === false) {
+			$output->writeln('<error>Could not transfer files</error>');
+			// Abort
+			return 1;
```
Comment (nickvergessen) [inline apps/files/lib/Command/TransferOwnership.php]: Is the real error logged?

Comment (ChristophWurst) [reply apps/files/lib/Command/TransferOwnership.php]: Couldn't find out. Some exceptions are thrown, some are caught.

Comment (ChristophWurst) [reply apps/files/lib/Command/TransferOwnership.php]: Maybe a storage export like @icewind1991 or @rullzer can tell?

Comment (rullzer) [reply apps/files/lib/Command/TransferOwnership.php]: I guess here it is not possible...

**Review by rullzer — APPROVED** [review]

**Comment by rullzer** [comment]
I guess this needs a rebase then :wink:

**Comment by ChristophWurst** [comment]
> I guess this needs a rebase then wink

Done. So much cleaner now :)

**Review by rullzer — APPROVED** [review]

**Comment by rullzer** [comment]
Review time!

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**